storage/replica.go

```diff
@@ -145,6 +149,10 @@ func updatesTimestampCache(r roachpb.Request) bool {
 	return updatesTimestampCacheMethods[m]
 }
 
+const replicaTooOld = "replica too old, discarding message"
+
+var errReplicaTooOld = grpc.Errorf(codes.Aborted, replicaTooOld)
```

Comment:
Maybe just create those when you need them. You're not using them as a sentinel, so the fact that we use both the error and the error message is a bit confusing.